### ISSUE TITLE
Lights are no longer controllable in v2.6.1.

### ISSUE THREAD

**Issue by ahnyerkeester** [hjdhjd/homebridge-myq#252]
**Describe The Problem:**
The lights function no longer works.
The garage door opener works fine.

**To Reproduce:**
When I tell Siri to turn off the lights she says the lights are off but they stay on.
Same thing with turning them on.

**Logs:**

```
1/25/2021, 1:37:16 PM [myQ] myQ API: https://account-devices-lamp.myq-cloud.com/api/v5.2/Accounts/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/lamps/CGXXXXXXXXXA/turnoff Error: 400 Bad Request
1/25/2021, 1:37:16 PM [myQ] myQ API: Unable to send the command to myQ servers. Acquiring a new access token.
1/25/2021, 1:37:16 PM [myQ] myQ API: https://account-devices-lamp.myq-cloud.com/api/v5.2/Accounts/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/lamps/CGXXXXXXXXX1/turnoff Error: 400 Bad Request
1/25/2021, 1:37:16 PM [myQ] myQ API: Unable to send the command to myQ servers. Acquiring a new access token.
1/25/2021, 1:38:10 PM [myQ] myQ API: Successfully connected to the myQ API.
```

**Homebridge Configuration:**

Nothings changed. 

**Environment:**

* **Homebridge Version:** 3.3.2
* **Node Version:** 12.14.1
* **Homebridge-myQ Plugin Version**: 2.6.1
* **Apple Device and iOS / macOS / iPadOS / tvOS Version:** iPhone iOS 14.3
* **Operating System and OS Version:** macMini (Late 2012) running macOS 10.15.7

**Comment by hjdhjd** [comment]
This is a known issue. It will be resolved when we better understand the new API. In the meantime, you can downgrade, if the previous version was working for you.

Locking this issue as well for future folks who run across this. When this issue is resolved, this issue will be updated and closed.

Thanks!

**Comment by hjdhjd** [comment]
Fixed in v2.6.2...enjoy!
